.travis.yml

```diff
@@ -1,10 +1,64 @@
 language: php
-         
+
+sudo: required
```

Comment:
I thought sudo is required for managing packages (I'm not sure). Should I try to [migrate to container-based infrastructure](https://docs.travis-ci.com/user/migrating-from-legacy/) by setting sudo to false?